bundle/edu.gemini.spdb.reports.collection/src/main/scala/edu/gemini/dbTools/timingwindowcheck/TimingWindowCheckMailer.scala

```diff
@@ -0,0 +1,35 @@
+package edu.gemini.dbTools.timingwindowcheck
+
+import edu.gemini.dbTools.mailer.{ Mailer, MailerType, ProgramAddresses }
+import edu.gemini.spModel.core.{ Site, SPProgramID }
+
+import scalaz._
+import Scalaz._
+import scalaz.effect.IO
+
+sealed abstract class TimingWindowCheckMailer(mailer: Mailer) {
+
+  def notifyPendingCheck(
+    pid:   SPProgramID,
+    to:    ProgramAddresses
+  ): Action[Unit] = {
+
+    val subject   =
+      s"${pid.stringValue} Timing Window Still Pending"
```

Comment:
Probably we'll want something like `"Timing window  closing in ${pid.stringValue}"`.  Science may be able to suggest better wording.